### PR TITLE
[SDPA-6233] Add new Display Options for Display Headings

### PR DESCRIPTION
**Pull request by mayurngondhkar**
**JIRA** - https://digital-engagement.atlassian.net/browse/SDPA-6233
           Parent Ticket - https://digital-engagement.atlassian.net/browse/SDPA-6190
* Updated text values
  * 'showH2' -> 'Show Heading 2s only.'
  * 'showH2AndH3' -> 'Show Heading 2s and heading 3s.'
* Added options
  * 'showH2AndAccordionTitle' -> 'Show Heading 2s and accordion titles.'
  * 'showH2AndH3AndAccordionTitle' -> 'Show Heading 2s, accordion titles and Heading 3s.'

### CODE REVIEW

**Comment by MdNadimHossain** [comment]
@mayurngondhkar can you have a quick look why the php unit test failing please. Also as this is a field settings change and you are updating the existing field settings values, just wondering if it is worth to give it a quick try on content-vic just to see how does it behave for exisiting contents.

**Comment by mayurngondhkar** [comment]
> @mayurngondhkar can you have a quick look why the php unit test failing please. Also as this is a field settings change and you are updating the existing field settings values, just wondering if it is worth to give it a quick try on content-vic just to see how does it behave for exisiting contents.

Hey @MdNadimHossain ,

Ticket created to address that issue https://digital-engagement.atlassian.net/browse/SDPA-6271?atlOrigin=eyJpIjoiYWNlOGEzNTI1ZDc3NGVhZTg0MTQ0OGIxODNlYzc3N2QiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

**Comment by vincent-gao** [comment]
hey @mayurngondhkar 
why the update hook resides in `tide_landing_page` module? should it be part of https://github.com/dpc-sdp/tide_core/pull/312/

File: tide_landing_page.install [hunk 1]
```diff
@@ -2438,3 +2438,23 @@ function tide_landing_page_update_8043() {
     $config->set('content.field_statistic_block.settings.default_paragraph_count', '2')->save();
   }
 }
+
+/**
+ * Update and Add new options for "Show table of contents" "Display heading".
+ */
+function tide_landing_page_update_8044() {
```
Comment (vincent-gao) [inline tide_landing_page.install:2445]: hey @mayurngondhkar 
why the update hook resides in `tide_landing_page` module? should it be part of https://github.com/dpc-sdp/tide_core/pull/312/

**Review by vincent-gao — CHANGES_REQUESTED** [review]

File: tide_landing_page.install (hunk 1 above)
Comment (mayurngondhkar) [reply tide_landing_page.install:2445]: Hey @vincent-gao ,

Yup,  I've moved it to tide_core 👍

**Pull request closed without merging**